h/static/styles/about-this-version-dialog.scss

```diff
@@ -1,23 +1,49 @@
+.about-this-version-dialog {
```

Comment:
![Guidelines](https://thisiswhylife.files.wordpress.com/2014/02/tumblr_lzwm2hkmgx1qhkwbs-1.gif?w=660)

The short answer is yes, though we're inconsistent about this elsewhere in the codebase at the moment.

The original BEM guideline was that a 'block' (the first part of the name before the `__`) is "something that is self contained" whereas an 'element' is "something that only makes sense within the context of its containing block". In that context it is sometimes not completely obvious what should be a 'block' vs. an 'element'

For our purposes, the important thing is to make it easy to find the styling that relates to a given component and be able to make changes without having unexpected effects elsewhere. In that context, having the name of the element in the markup, the block name in the styles and the name of the relevant sources files all match is helpful.